### DISCUSSION TITLE
Gestion des images dans les blocs de description d'un flux

### DISCUSSION THREAD

**Discussion by hadhub** [dbarzin/mercator#836] [Q&A] [answered]
Bonjour,

Pour mieux expliquer un flux que je dois cartographier, je cherche à ajouter une image format .png au bloc description.
Lorsque j'importe l'image j'ai la fenêtre de sélection qui s'ouvre, néanmoins je n'ai pas de retour visuel dessus, ni de message d'erreur.

Est-ce un bug que je dois report côté issu ? ou la fonctionnalité n'est pas encore totalement intégrée ?

Bien à vous, 

Hadrien CHAUDER

**Comment by dbarzin** [comment]
Les blocks description ne permettent pas d'ajouter des images. Cette limitation a été introduite pour permettre d'exporter les descriptions dans une feuille de calcul.

**Reply by hadhub** [reply, marked as answer]
D'accord, merci pour votre réponse
